### ISSUE TITLE
In Nook - after "go to" opened page remains white

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#360]
```
In Nook Simple Touch rooted and Ebookdroid 1.5.6 (1.5.2 and 1.5.5 also) :

1. When you move to another page when a high magnification, a new page remains 
white and only after little Pinch back to shows the text.

What is Ebookdroid optimized version good for NST?
```

Original issue reported on code.google.com by `gold...@gmail.com` on 27 Oct 2012 at 9:12

**Comment by GoogleCodeExporter** [comment]
```
It means that OutOfMemory error occurred on first decoding.
After that memory has been cleared.
So only when you move or pinch book, the current page is re-decoding.
Possibly it will be fixed in the future
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 2 Nov 2012 at 10:25
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
NST and other 2.1 and older devices support is discontinued.

Try turn on "bitmap allocation hack" and reusing textures.
```

Original comment by `Andrei.K...@gmail.com` on 16 Nov 2012 at 2:42
- Changed state: **WontFix**
